2025-02-07 18:19:09,433 - src.textSummarizer.utils.common - INFO - common.py - 37 - Directory created: artifacts
2025-02-07 18:20:52,539 - __main__ - INFO - 3106384179.py - 9 - Crating the directory artifacts/data_ingestion/summarizer-data.zip
2025-02-07 18:20:52,917 - __main__ - ERROR - 3106384179.py - 19 - Error downloading data from https://github.com/krishnaik06/datasets/raw/refs/heads/main/summarizer-data.zip
2025-02-07 18:20:52,918 - __main__ - ERROR - 3106384179.py - 20 - [Errno 21] Is a directory: 'artifacts/data_ingestion/summarizer-data.zip'
2025-02-07 18:22:13,687 - __main__ - INFO - 3106384179.py - 9 - Crating the directory artifacts/data_ingestion/summarizer-data.zip
2025-02-07 18:22:13,727 - __main__ - ERROR - 3106384179.py - 19 - Error downloading data from https://github.com/krishnaik06/datasets/raw/refs/heads/main/summarizer-data.zip
2025-02-07 18:22:13,728 - __main__ - ERROR - 3106384179.py - 20 - [Errno 21] Is a directory: 'artifacts/data_ingestion/summarizer-data.zip'
2025-02-07 18:23:47,859 - __main__ - INFO - 2354293849.py - 20 - Downloading data from https://github.com/krishnaik06/datasets/raw/refs/heads/main/summarizer-data.zip
2025-02-07 18:23:48,169 - __main__ - ERROR - 2354293849.py - 41 - Error extracting zip file to artifacts/data_ingestion
2025-02-07 18:23:48,170 - __main__ - ERROR - 2354293849.py - 42 - [Errno 21] Is a directory: 'artifacts/data_ingestion/summarizer-data.zip'
2025-02-07 18:24:05,341 - __main__ - INFO - 2354293849.py - 25 - File artifacts/data_ingestion/summarizer-data.zip/summarizer-data.zip already exists
2025-02-07 18:24:05,343 - __main__ - ERROR - 2354293849.py - 41 - Error extracting zip file to artifacts/data_ingestion
2025-02-07 18:24:05,344 - __main__ - ERROR - 2354293849.py - 42 - [Errno 21] Is a directory: 'artifacts/data_ingestion/summarizer-data.zip'
2025-02-07 18:25:07,998 - src.textSummarizer.utils.common - ERROR - common.py - 26 - Error reading yaml file: [Errno 2] No such file or directory: 'config/config.yaml'
2025-02-07 18:25:11,820 - src.textSummarizer.utils.common - ERROR - common.py - 26 - Error reading yaml file: [Errno 2] No such file or directory: 'config/config.yaml'
2025-02-07 18:25:27,959 - __main__ - INFO - 2573542362.py - 25 - File artifacts/data_ingestion/summarizer-data.zip/summarizer-data.zip already exists
2025-02-07 18:25:28,091 - __main__ - INFO - 2573542362.py - 43 - Extracting zip file to artifacts/data_ingestion
2025-02-07 18:37:17,831 - __main__ - INFO - main.py - 9 - 🏃🏼‍➡️🏃🏼‍➡️🏃🏼‍➡️ Starting Data Ingestion Stage 🏃🏼‍➡️🏃🏼‍➡️🏃🏼‍➡️
2025-02-07 18:37:17,833 - src.textSummarizer.components.data_ingestion - INFO - data_ingestion.py - 30 - File artifacts/data_ingestion/summarizer-data.zip/summarizer-data.zip already exists
2025-02-07 18:37:17,965 - src.textSummarizer.components.data_ingestion - INFO - data_ingestion.py - 48 - Extracting zip file to artifacts/data_ingestion
2025-02-07 18:37:17,965 - __main__ - INFO - main.py - 12 - ✅✅✅ Completed Data Ingestion Stage ✅✅✅
2025-02-07 18:38:29,315 - __main__ - INFO - main.py - 9 - 🏃🏼‍➡️🏃🏼‍➡️🏃🏼‍➡️ Starting Data Ingestion Stage 🏃🏼‍➡️🏃🏼‍➡️🏃🏼‍➡️
2025-02-07 18:38:29,317 - src.textSummarizer.components.data_ingestion - INFO - data_ingestion.py - 18 - Creating the directory artifacts/data_ingestion/summarizer-data.zip
2025-02-07 18:38:29,317 - src.textSummarizer.components.data_ingestion - INFO - data_ingestion.py - 25 - Downloading data from https://github.com/krishnaik06/datasets/raw/refs/heads/main/summarizer-data.zip
2025-02-07 18:38:29,822 - src.textSummarizer.components.data_ingestion - INFO - data_ingestion.py - 48 - Extracting zip file to artifacts/data_ingestion
2025-02-07 18:38:29,822 - __main__ - INFO - main.py - 12 - ✅✅✅ Completed Data Ingestion Stage ✅✅✅
